### PR TITLE
Jerie/home page stylings for pr

### PR DESCRIPTION
**Pull request by jmacap**
## What?
This is a very minor PR request that improves the look of the home page.

## Why?
Given that demo presentation is quite soon, I thought it might be useful for demonstrators to have a sense of what our app would look like at it's final stage and hopefully get some feed back for it.

## How?
Removal of transcript bar + addition of nice banner image.

## Anything else?
The work is all concentrated in the home page which would make it easier to review. The last three commits are of primary concern to review, there is also an addition to static content on the privacy page which fleshes out the website a little bit more though no programming logic was implemented here, just words. Commits beyond [58e0e03] were merges to update the branch and additions/deletions that are no longer relevant to the project.

### CODE REVIEW

File: FrontEnd/air-quality-app/src/pages/Privacy.jsx
Comment (Naveen-Jaisankar) [inline FrontEnd/air-quality-app/src/pages/Privacy.jsx:1]: Privacy policy

Sample : https://support.only.com/hc/en-ie/articles/10280962923281-Privacy-Policy

1. Should always have Published on and Last updated on 
2. Content
	- Privacy statement
	- Use of personal data
	- Technical term used - AQI and so on
	- Feedback form 
3. Go back to previous page Functionality

File: FrontEnd/air-quality-app/src/pages/Home.jsx
Comment (Naveen-Jaisankar) [inline FrontEnd/air-quality-app/src/pages/Home.jsx:1]: Please add footer in the landing page

**Review by Naveen-Jaisankar — CHANGES_REQUESTED** [review]

File: FrontEnd/air-quality-app/src/pages/Privacy.jsx
Comment (jmacap) [reply FrontEnd/air-quality-app/src/pages/Privacy.jsx:1]: Thank you, this is really helpful! Will update now

File: FrontEnd/air-quality-app/src/pages/Home.jsx
Comment (jmacap) [reply FrontEnd/air-quality-app/src/pages/Home.jsx:1]: Ah pardon me will add now also

**Pull request merged**